samples/Server/src/routes/createRoom.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import { getResourceConnectionString } from '../lib/envHelper';
+import { RoomsClient, CreateRoomOptions } from '@azure/communication-rooms';
+import { createUser } from '../lib/identityClient';
+
+const router = express.Router();
+
+/**
+ * route: /createRoom/
+ *
+ * purpose: Create a new room.
+ *
+ * @returns The new roomId as string
+ */
+
+router.post('/', async function (req, res, next) {
+  // create RoomsClient
+  const roomsClient: RoomsClient = new RoomsClient(getResourceConnectionString());
+
+  const validFrom = new Date();
+  const validUntil = new Date(validFrom.getTime() + 5 * 60 * 1000);
```

Comment:
can we have a comment for what this time length is representing? I am guessing it means 

the room is available from now till now + 5 hrs?